josm-presets/de-signals-eso.xml

```diff
@@ -145,6 +145,22 @@ See https://wiki.openstreetmap.org/wiki/OpenRailwayMap for details.
 						display_values="substitute signal (Zs 1);cautioness signal (Zs 7);M Board (Zs 12)"
 						de.display_values="Ersatzsignal (Zs 1);Vorsichtsignal (Zs 7);M-Tafel (Zs 12)"
 						/>
+					<multiselect key="railway:signal:traversable:type"
+					    text="Traversable: action at stop"
+					    de.text="Mastschild: Verhalten bei Halt"
+					    values="written_default_speed;dictated_default_speed;always_on_sight;on_no_contact_on_sight"
+					    display_values="written_default_speed (traversable red/red-white/yellow-white);dictated_default_speed (traversable red/red-white);always_on_sight (traversable black-white);on_no_contact_on_sight (traversable yellow-white)"
+					    de.display_values="Mastschild rot / rot-weiß / gelb-weiß;Mastschild rot / rot-weiß;Mastschild schwarz-ẁeiß;Mastschild gelb-weiß"
```

Comment:
You have funny "ẁ" characters in every of these lines.

Reply:
I haven't really seen them, but they should be replaced. Adiotionally replaced some spaces through the vim ^I tab stop. Hope it's better now...